tests/bootstrap.php

```diff
@@ -13,10 +13,6 @@
 $loader->add('Jackalope\\Test', __DIR__);
 $loader->add('Jackalope', __DIR__ . '/../vendor/jackalope/jackalope/tests');
```

Comment:
only for the library itself afaik. i think jackalope-doctrine-dbal does
not use the autoload-dev from its dependencies like jackalope/jackalope